src/main/java/seedu/address/logic/commands/DeleteBuyerCommand.java

```diff
@@ -0,0 +1,82 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+
+import java.util.List;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+/**
+ * Represents a command to delete a buyer in the buyer management system.
+ */
+public class DeleteBuyerCommand extends Command {
+    /** The command word for this specific action. */
+    public static final String COMMAND_WORD = "deletebuyer";
+    public static final String MESSAGE_ARGUMENTS = "phoneNumber: %1$s";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the buyer identified by the phone number used in the displayed person list.\n"
+            + "Parameters: phone number (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " " + PREFIX_PHONE + "81621234";
+    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
```

Comment:
should we try changing all the references to `Person` to either `Buyer`/ `Seller` specifically? I implemented it as such in the `addbuyer`/ `addseller` commands